docs/components/icons/elements.md

```diff
@@ -0,0 +1,11 @@
+## Import
+
+```js
+import '@warp-ds/icons/elements/icon-bag-16';
+```
+
+### Usage
+
+```html
+  <w-icon-bag16 class="s-icon"/>
```

Comment:
I just wanted to add it here to visualize that we have icon classes. Do not know if we should add it to the components. Icons will get current color right otherwise, right?

Reply:
Yes, icons will get current color from the wrapper. I'm unsure if the above example will be understood as "this is what you can do". At least when I saw it, I understood it as "this is what you should do when using icons". But maybe it's just me 😊  Could be worth mentioning somewhere here that icons default to `currentColor` and that colors can be changed using semantic color classes for icons listed here: https://warp-ds.github.io/css-docs/icon-color#icon-color.

Reply:
That is what I asked here `Do we want to link to the css docs from here so consumers can find icon colors easier?` in the description. I'll add a link and remove the usage of the class

Reply:
I was planning to answer questions from the PR description next ;) And yes, I think that it would be good to link to the css docs from this page.